packages/server/src/workers/subscription.ts

```diff
@@ -157,6 +163,9 @@ async function checkAccessPolicy(resource: Resource, project: Project, subscript
             `[Subscription Access Policy]: Access Policy not satisfied on '${resourceReference}' for '${subReference}'`,
             { subscription: subReference, project: projectReference, accessPolicy }
           );
+          if (subscription.channel.type === 'websocket') {
```

Comment:
Consolidate with a single ` if (subscription.channel.type === 'websocket') {}` at the end. 

This will make it easier to remove this case later